docs/reference/permission.rst

```diff
@@ -1,26 +1,125 @@
 .. _permission:
 
-Permission
-##########
+Permissions
+###########
 
+*Cliquet* provides a mechanism to handle authorization on the stored objects.
 
-Redis
-=====
+Glossary
+========
 
-.. autoclass:: cliquet.permission.redis.Redis
+Authorization isn't complicated, but requires the introduction of a few terms
+so that explanations are easier to follow:
 
+**Object**:
+    The data that is stored into *Cliquet*. Objects usually match
+    the resources you defined; For one resource there are two objects: resource's
+    collection and resource's records.
+**Principal**:
+    An entity that can be authenticated. Principals can be individual people,
+    computers, services, or any group of such things.
+**Permission**:
+    An action that can be authorized or denied. *read*, *write*, *create* are
+    permissions.
+**Access Control Entity (ACE)**:
+    An association of a principal, an object and a permission. For instance,
+    (Alexis, article, write).
+**Access Control List (ACL)**:
+    A list of Access Control Entities (ACE)
 
-Memory
-======
+Overview
+========
 
-.. autoclass:: cliquet.permission.memory.Memory
+By default, the resources defined by *Cliquet* are public. but it is also
+possible to define *protected resources*, which will required the user to have
+access to the requested resource.
 
+.. code-block:: python
 
-API
-===
+    from cliquet import authorization
+    from cliquet import resource
 
-Implementing a custom permission backend consists in implementating
-the following interface:
+
+    @resource.register(factory=authorization.RouteFactory)
+    class Toadstool(resource.ProtectedResource):
+        mapping = MushroomSchema()
```

Comment:
I realize while reading these docs, that we missed something :) 
If we want to introduce record sharing via permissions, the ``self.collection.parent_id`` must be set to something else than ``self.request.authenticated_userid``. Otherwise resources records are always isolated by user.